src/api/Elastic.Documentation.Api.Lambda/Program.cs

```diff
@@ -5,20 +5,22 @@
 using System.Text.Json.Serialization;
 using Amazon.Lambda.APIGatewayEvents;
 using Amazon.Lambda.Serialization.SystemTextJson;
+using Elastic.Documentation.Api.Core.AskAi;
 using Elastic.Documentation.Api.Infrastructure;
 
 var builder = WebApplication.CreateSlimBuilder(args);
 
 builder.Services.AddAWSLambdaHosting(LambdaEventSource.RestApi, new SourceGeneratorLambdaJsonSerializer<LambdaJsonSerializerContext>());
-builder.Services.AddElasticDocsApiUsecases(Environment.GetEnvironmentVariable("APP_ENVIRONMENT"));
+builder.Services.AddElasticDocsApiUsecases(Environment.GetEnvironmentVariable("ENVIRONMENT") ?? "dev");
 
 var app = builder.Build();
 
-var v1 = app.MapGroup("/v1");
+var v1 = app.MapGroup("/docs/_api/v1");
```

Comment:
We need a few reserved namespaces in the markdown build :). I have `/docs/api` for the OpenAPI too.